packages/transition-backend/src/models/db/jobs.db.queries.ts

```diff
@@ -49,6 +50,19 @@ const collection = async (
             if (options.jobType !== undefined) {
                 query.where('name', options.jobType);
             }
+            if (options.statuses !== undefined) {
+                const statuses = options.statuses;
+                if (statuses.length === 1) {
+                    query.where('status', options.statuses[0]);
+                } else if (statuses.length > 1) {
+                    query.where(function () {
+                        this.where('status', statuses[0] as JobStatus);
```

Comment:
I don't know much about knex, but instead of orWhere, is it possible to do a WHERE iN (LIST of status)

I would think that would be more optimal. I don't know if the optimisation is done automatically)